com.wps.WPS.appdata.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+ <id>com.wps.WPS</id>
+ <metadata_license>CC0-1.0</metadata_license>
+ <project_license>LicenseRef-proprietary=http://wps-community.org/license.md</project_license>
+ <name>WPS Office</name>
+ <summary>WPS Office Suite</summary>
+ <description>
+  WPS Office including Writer, Presentation
+  and Spreadsheets, is a powerful office suite, which is able to process
+  word file, produce wonderful slides, and analyze data as well. It is
+  deeply compatible with all of the latest Microsoft Office file formats.
+  It can easily open and read the documents created with Microsoft Office.
+ </description>
+ <launchable type="desktop-id">com.wps.WPS-wps.desktop</launchable>
+ <url type="homepage">https://www.wps.com/linux</url>
+ <url type="bugtracker">http://community.wps.cn/bug/login_page.php</url>
+ <url type="donation">http://wps-community.org/donate</url>
+ <url type="faq">http://wps-community.org/faq</url>
+ <url type="help">http://community.wps.cn/wiki</url>
+ <url type="translate">https://github.com/wps-community/wps_i18n</url>
+ <developer_name>Kingsoft Office Corporation</developer_name>
+ <update_contact>wps_linux_at_kingsoft.com</update_contact>
+ <releases>
+  <release
+    version="10.1.0.5707~a21"
+    date="2017-06-15"/>
+ </releases>
```

Comment:
we need to add OARS here - just the simple 1.1 version should be fine (and mostly no's!)